change/@fluentui-react-avatar-f0ea8fa4-052d-4a92-a148-acfeb327bc3f.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the 27 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=26811&commitId=a6874d153bebc8c8e35c4cb7c5a44baa1df2ae68&env=prod)

✅ There was 0 screenshots added, 0 screenshots removed, 1821 screenshots unchanged, 0 screenshots with different dimensions and 27 screenshots with visible difference.


  <details><summary><b>unknown</b> 27 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|AvatarGroup Converged.basic - Dark Mode.default.chromium.png|29789| Changed|
|AvatarGroup Converged.basic - High Contrast.default.chromium.png|55643| Changed|
|AvatarGroup Converged.basic.default.chromium.png|29248| Changed|
|AvatarGroup Converged.layoutPie - Dark Mode.default.chromium.png|9808| Changed|
|AvatarGroup Converged.layoutPie - High Contrast.default.chromium.png|21623| Changed|
|AvatarGroup Converged.layoutPie - RTL.default.chromium.png|12998| Changed|
|AvatarGroup Converged.layoutPie-1 - Dark Mode.default.chromium.png|2493| Changed|
|AvatarGroup Converged.layoutPie-1 - High Contrast.default.chromium.png|2421| Changed|
|AvatarGroup Converged.layoutPie-1 - RTL.default.chromium.png|3113| Changed|
|AvatarGroup Converged.layoutPie-1.default.chromium.png|3111| Changed|
|AvatarGroup Converged.layoutPie-2 - Dark Mode.default.chromium.png|2526| Changed|
|AvatarGroup Converged.layoutPie-2 - High Contrast.default.chromium.png|2389| Changed|
|AvatarGroup Converged.layoutPie-2 - RTL.default.chromium.png|3084| Changed|
|AvatarGroup Converged.layoutPie-2.default.chromium.png|3099| Changed|
|AvatarGroup Converged.layoutPie.default.chromium.png|12992| Changed|
|AvatarGroup Converged.layoutStack - Dark Mode.default.chromium.png|27709| Changed|
|AvatarGroup Converged.layoutStack - High Contrast.default.chromium.png|49959| Changed|
|AvatarGroup Converged.layoutStack.default.chromium.png|26665| Changed|
|AvatarGroup Converged.overflowContent - Dark Mode.chromium.png|7306| Changed|
|AvatarGroup Converged.overflowContent - Dark Mode.popoverContentOpen.chromium.png|49884| Changed|
|AvatarGroup Converged.overflowContent - High Contrast.chromium.png|8965| Changed|
|AvatarGroup Converged.overflowContent - High Contrast.popoverContentOpen.chromium.png|59364| Changed|
|AvatarGroup Converged.overflowContent.chromium.png|1003| Changed|
|AvatarGroup Converged.overflowContent.popoverContentOpen.chromium.png|6275| Changed|
|AvatarGroup Converged.overflowIndicator - Dark Mode.default.chromium.png|28163| Changed|
|AvatarGroup Converged.overflowIndicator - High Contrast.default.chromium.png|53717| Changed|
|AvatarGroup Converged.overflowIndicator.default.chromium.png|27585| Changed|

</details>


<div id="vrtCommentfluentuiv9"/>